src/Controls/src/Core/Platform/Windows/CollectionView/ItemContentControl.cs

```diff
@@ -325,5 +325,7 @@ bool CanMeasureContent(FrameworkElement frameworkElement)
 
 			return true;
 		}
+
+		internal VisualElement GetVisualElement() => _renderer?.VirtualView as VisualElement;
```

Comment:
Is this not `_visualElement`? I see it is set to that in `void Realize()`.

Reply:
Oh would you look at that. You're right! I was looking higher up in `UpdateIsSelected()` and there `renderer?.VirtualView` was used. I made the fix and this PR still works 👍